modules/virt-configuring-vm-dpdk.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-vm-dpdk_{context}"]
+= Configuring a virtual machine for DPDK workloads
+
+You can run Data Packet Development Kit (DPDK) workloads on virtual machines (VMs) to achieve lower latency and higher throughput for faster packet processing in the user space. DPDK uses the SR-IOV network for hardware-based I/O sharing.
+
+.Prerequisites
+* Your cluster is configured to run DPDK workloads.
+* You have created and configured the project in which the VM will run.
+
+.Procedure
+. Edit the `VirtualMachine` manifest to include information about the SR-IOV network interface, CPU topology, CRI-O annotations, and huge pages:
++
+.Example `VirtualMachine` manifest
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: rhel-dpdk-vm
+spec:
+  running: true
+  template:
+    metadata:
+      annotations:
+        cpu-load-balancing.crio.io: disable <1>
+        cpu-quota.crio.io: disable <2>
+        irq-load-balancing.crio.io: disable <3>
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/worker-dpdk: "" <4>
+      domain:
+        cpu:
+          sockets: 1 <5>
+          cores: 5 <6>
+          threads: 2
+          dedicatedCpuPlacement: true
+          isolateEmulatorThread: true
+        interfaces:
+          - masquerade: {}
+            name: default
+          - model: virtio
+            name: nic-east
+            pciAddress: '0000:07:00.0'
+            sriov: {}
+          networkInterfaceMultiqueue: true
+          rng: {}
+      memory:
+        hugepages:
+          pageSize: 1Gi <7>
+      resources:
+        requests:
+          memory: 8Gi
+      networks:
+        - name: default
+          pod: {}
+        - multus:
+            networkName: dpdk-net <8>
+          name: nic-east
+...
+----
+<1> This annotation specifies that load balancing is disabled for CPUs that are used by the container.
+<2> This annotation specifies that the CPU quota is disabled for CPUs that are used by the container.
+<3> This annotation specifies that Interrupt ReQuest (IRQ) load balancing is disabled for CPUs that are used by the container.
```

Comment:
"ReQuest" - I understand that it is written this way in order to clarify the IRQ abbreviation, but are you sure it is valid to write it this way (i.e. CamelCase inside a single word)?
If this is indeed a common practice in technical writing, then I apologize in advance for the noise.

Reply:
Thanks @yossisegev. Our style guidelines do say " When you spell out an abbreviation, do not capitalize letters in the middle of a word to show the origin of an initialism."

I will update the text.